src/utils.rs

```diff
@@ -4,19 +4,6 @@
 
 use ffi;
 use translate::*;
-use UserDirectory;
-
-pub fn get_application_name() -> Option<String> {
```

Comment:
But previously it was available as e.g. utils::get_application_name(). Now it would be auto::functions::get_application_name() (and the auto module is not pub-re-exported so that won't work)